frontend/src/components/UserStatBlock.svelte

```diff
@@ -53,12 +64,14 @@
 </style>
 
 <li class="user-stat-block" in:blur>
-  <p class="rank">#{rank}</p>
-  <img class="avatar" src="{userStats.avatarUrl}" alt="{userStats.user}'s Profile Image">
-  <p class="name">{userStats.user}</p>
-  <div class="stat-block">
-    <p><span class="emoji-stat-light">⚔️️️️️️</span>+{userStats.additions}&emsp;-{userStats.deletions}</p>
-    <p><span class="emoji-stat-light">💾</span>{userStats.commits} {userStats.commits === 1 ? 'Commit' : 'Commits'}</p>
-    <p>{userStats.userId}</p>
-  </div>
+  <a href="https://github.com/{userStats.user}" target="_blank">
+    <p class="rank">#{rank}</p>
+    <img class="avatar" src="{userStats.avatarUrl}" alt="{userStats.user}'s Profile Image">
+    <p class="name">{userStats.user}</p>
+    <div class="stat-block">
+      <p><span class="emoji-stat-light">⚔️️️️️️</span>+{userStats.additions}&emsp;-{userStats.deletions}</p>
+      <p><span class="emoji-stat-light">💾</span>{userStats.commits} {userStats.commits === 1 ? 'Commit' : 'Commits'}</p>
+      <p>{userStats.userId}</p>
+    </div>
+  </a>
```

Comment:
Can you change this to be the user's name (and maybe the avatar as well) instead of the whole card?

```suggestion
<p class="rank">#{rank}</p>
<img class="avatar" src="{userStats.avatarUrl}" alt="{userStats.user}'s Profile Image">
<p class="name"><a href="https://github.com/{userStats.user}" target="_blank">{userStats.user}</a></p>
<div class="stat-block">
  <p><span class="emoji-stat-light">⚔️️️️️️</span>+{userStats.additions}&emsp;-{userStats.deletions}</p>
  <p><span class="emoji-stat-light">💾</span>{userStats.commits} {userStats.commits === 1 ? 'Commit' : 'Commits'}</p>
</div>
```

Reply:
Is the suggestion good to go? Or is the styling off and needs to be fixed? Will work on it tomorrow afterthought after work 😁 should the image be linked aswell?

Reply:
The suggestion was more an example... So probably not :sweat_smile: 

Up to you on the image - if it you feel it is intuitive then :100: do it.

Reply:
What about the hover transition? I think since the whole card is not clickable, animating the whole thing seems wrong. Any suggestions? 😁

Reply:
Hmm.. :thinking: Fair point. That does seem a bit counter-intuitive. All I can think of is to play around with it, and see how it feels :smiley: